### PR TITLE
Tidy up WizardBasePanel

### PR DESCRIPTION
**Pull request by ckelleyRH**
* Don't return unnecessary Boolean literals
* Use try-with-resources to take advantage of Autoclosable

### CODE REVIEW

File: base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java [hunk 1]
```diff
@@ -154,8 +160,8 @@ public String getErrorMessage(WizardInfo wizardInfo) {
 
     public boolean send(String host, int port, String servlet, String rawData,
       WizardInfo wizardInfo) {
-        try {
-            Socket socket = new Socket(host, port);
+        try (Socket socket = new Socket(host, port);
```
Comment (c-dorney) [inline base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java:163]: This is cool

File: base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java [hunk 2]
```diff
@@ -193,13 +197,8 @@ public boolean send(String host, int port, String servlet, String rawData,
...
             }
-            bstream.close();
             String sendStatus = (String)wizardInfo.get("NMC_STATUS");
-            if (sendStatus.equals("0")) {
-                return true;
-            } else {
-                return false;
-            }
+            return sendStatus.equals("0");
         } catch (Exception e) {
```
Comment (c-dorney) [inline base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java:202]: Something should probably get logged here (same with in other method)

File: base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java (hunk 1 above)
Comment (ckelleyRH) [reply base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java:163]: I am a fan of `try-with-multiple-resources` although I admit it sends me to Stack Overflow every time. I always forget that you separate the resources with a semicolon and not a comma, "Why won't you compile" etc :smile:

File: base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java (hunk 2 above)
Comment (ckelleyRH) [reply base/console/src/main/java/com/netscape/admin/certsrv/wizard/WizardBasePanel.java:203]: I have added in a print of the `sendStatus` value before returning it, The class doesn't define a logger, so I didn't introduce one.

**Comment by ckelleyRH** [comment]
@c-dorney I noticed that there were potential NPE opportunities in `getErrorMessage(WizardInfo wizardInfo)` - I removed the check for the string being empty because most of the calling code checks if the string is empty and provides a default message if it is. I don't see the reason for the additional check, so now we always return the string provided it is not `null`.

I think we should return the empty string instead of `null` anyway but that's becoming a bit more of a refactor than I want to do in this changeset.

**Review by edewata — APPROVED** [review]
LGTM, but just a reminder that we've deprecated console.

**Comment by ckelleyRH** [comment]
Good point!

**Pull request merged**